Upgrade to Guacamole 1.5.1

jsf9k opened the pull request:
## 🗣 Description ##

This pull request:
- Upgrades the Docker composition to use the recently-released  `guacamole/guacd:1.5.1` and `guacamole/guacamole:1.5.1` Docker images
- Upgrades the Docker composition to use a more recent version of the PostgreSQL Docker image.

## 💭 Motivation and context ##

Guacamole 1.5.1 is the latest release, and is therefore likely to be more performant and secure.

The ability to upgrade to a more recent version of the PostgreSQL Docker image is a side effect of the Guacamole Docker image upgrade.

Resolves #37.

## 🧪 Testing ##

I deployed these changes to our COOL staging environment and verified that they functioned as expected.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [x] Finalize version.
- [x] Bump to version `1.0.0`.

## ✅ Post-merge checklist ##

- [x] Create a release.

jsf9k committed:
Bump version from 0.1.6 to 0.1.7

jsf9k committed:
Bump version from 0.1.7 to 0.1.7-rc.1

jsf9k committed:
Upgrade to version 1.5.0 of the guacamole and guacd containers

dav3r commented on src/version.txt:
```diff
@@ -1 +1 @@
-__version__ = "0.1.6"
+__version__ = "0.1.7-rc.1"
```
I feel like we can probably move this repo to a version 1.0.0 at this point, don't you think?

jsf9k replied:
I will bump the version to `1.0.0` after I finalize the version upon approval.

jsf9k replied:
It has been done.  @dav3r, could you resolve this conversation?

dav3r replied:
Resolved! 👍

jsf9k committed:
Fix one place where I neglected to update the guacamole container

jsf9k committed:
Bump version from 0.1.7-rc.1 to 0.1.7-rc.2

jsf9k commented:
We will need to wait for the release of version 1.5.1 because we are encountering [this bug](https://issues.apache.org/jira/projects/GUACAMOLE/issues/GUACAMOLE-1741?filter=allissues).  It appears that [version 1.5.1 should be released any day now](https://issues.apache.org/jira/projects/GUACAMOLE/issues/GUACAMOLE-1752?filter=allissues).

jsf9k committed:
Upgrade to version 1.5.1 of the official Guacamole Docker container

Version 1.5.1 fixes a bug that was causing 1.5.0 to fail for us:
https://issues.apache.org/jira/projects/GUACAMOLE/issues/GUACAMOLE-1741

jsf9k committed:
Bump version from 0.1.7-rc.2 to 0.1.7-rc.3

jsf9k committed:
Upgrade to the latest version of the PostgreSQL Docker image

jsf9k committed:
Bump version from 0.1.7-rc.3 to 0.1.7-rc.4

jsf9k commented:
> We will need to wait for the release of version 1.5.1 because we are encountering [this bug](https://issues.apache.org/jira/projects/GUACAMOLE/issues/GUACAMOLE-1741?filter=allissues). It appears that [version 1.5.1 should be released any day now](https://issues.apache.org/jira/projects/GUACAMOLE/issues/GUACAMOLE-1752?filter=allissues).

Version 1.5.1 has been released!

mcdonnnj reviewed (approved):
:shipit:

dav3r reviewed (approved):
🎉

jsf9k committed:
Finalize version from 0.1.7-rc.4 to 0.1.7

jsf9k committed:
Bump version from 0.1.7 to 1.0.0
